config/environment.default.js

```diff
@@ -10,10 +10,10 @@ module.exports = {
   // The REST API server settings.
   rest: {
     ssl: true,
-    host: 'dspace7.4science.it',
+    host: 'dspace7-entities.atmire.com',
```

Comment:
I understand the testing purpose, but I think this default file should be stuck to more default configurations like **localhost**.

Reply:
@paulo-graca : I agree for the final release. However, for the ongoing development of the Angular UI, we've decided to *default it* to point at our public demo site.  That makes PRs and code much easier to test because we don't require testers install the REST API locally (or figure out how to change this setting manually).

So, the plan is that the final DSpace 7 release *will have this set to localhost*.  However, until then, we will keep it set to the public demo site for easy testing/development.